### PR TITLE
fix: dockerfile.backend build process fixed so pycharm test running works

### PR DESCRIPTION
**Pull request by Bento007**
…

Signed-off-by: Trent Smith <trent.smith@chanzuckerberg.com>

- #TICKET_NUMBER

### Reviewers
**Functional:** @danieljhegeman 

**Readability:** 

---

## Changes
- modify dockerfile.backend build process fixed so pycharm test running works.

### CODE REVIEW

**Comment by danieljhegeman** [comment]
Interesting - why was this failing with Pycharm?

File: docker-compose.yml [hunk 1]
```diff
@@ -245,7 +245,6 @@ services:
         - HAPPY_BRANCH
         - HAPPY_TAG
     restart: always
-    command: [ "./container_init.sh" ]
```
Comment (danieljhegeman) [inline docker-compose.yml:248]: yeah this was superfluous given that a `dockerfile` is defined above right?

**Comment by Bento007** [comment]
Something about not being able to find "./container_init.sh"

**Comment by Bento007** [comment]
hold on maybe this does work fine. I was running this with the redesigned branch it was failing, but it seems to be working now on main.

**Comment by danieljhegeman** [comment]
Ah yes you might've needed to rebase. It's possible you were working off of `1220be39` on main but you needed my subsequent commit `8e9dc8ed` which followed a few hours later to fix exactly that error - not able to find `container_init.sh`

My fault on that. Unlucky on the timing. 🙋‍♂️ 😕 😞

**Comment by Bento007** [comment]
change not needed

**Pull request closed without merging**